cluster/swarm/utils.go

```diff
@@ -33,3 +34,13 @@ func convertMapToKVStrings(values map[string]*string) []string {
 	}
 	return result
 }
+
+var imageEngineOSErrorPattern = regexp.MustCompile(`cannot load (.+) image on (.+)`)
```

Comment:
What do you mean too general? Are you worried that this will catch other errors besides the ones we care about?